modules/services/podman-linux/default.nix

```diff
@@ -1,17 +1,99 @@
 { config, pkgs, lib, ... }:
-
-{
+let
+  cfg = config.services.podman;
+  toml = pkgs.formats.toml { };
+in {
   meta.maintainers = with lib.hm.maintainers; [ bamhm182 n-hass ];
 
   imports =
     [ ./containers.nix ./install-quadlet.nix ./networks.nix ./services.nix ];
 
   options.services.podman = {
     enable = lib.mkEnableOption "Podman, a daemonless container engine";
+
+    config = {
```

Comment:
I think this is probably best renamed to `settings` to avoid the overloaded term `config`. It's not quite an RFC-42 style settings but I think it is close enough.

Reply:
Yea, I agree. Done.